mne_bids/write.py

```diff
@@ -1155,10 +1222,31 @@ def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
     _participants_json(participants_json_fname, True, verbose)
     _scans_tsv(raw, op.join(kind, bids_fname), scans_fname, overwrite, verbose)
 
-    # TODO: Implement coordystem.json and electrodes.tsv for EEG and  iEEG
+    # TODO: Implement coordystem.json and electrodes.tsv for EEG
     if kind == 'meg' and not emptyroom:
         _coordsystem_json(raw, unit, orient, manufacturer, coordsystem_fname,
-                          overwrite, verbose)
+                          kind, overwrite, verbose)
+    elif kind == 'ieeg':
+        electrodes_fname = make_bids_basename(
+            subject=subject_id, session=session_id, acquisition=acquisition,
+            suffix='electrodes.tsv', prefix=data_path)
+
+        # We only write iEEG electrodes.tsv and accompanying coordsystem.json
+        # if we have an available DigMontage
+        if raw.info['dig'] is not None:
+            coords = _extract_landmarks(raw.info['dig'])
+            if set(['RPA', 'NAS', 'LPA']) == set(list(coords.keys())):
+                # Rescale to MNE-Python "head" coord system, which is the
+                # "ElektaNeuromag" system, and equivalent to "CapTrak"
```

Comment:
also mention that "ElektaNeuromag" and "CapTrak" are BIDS names just as you mention that "head" is MNE-Python nomenclature.